python/constraints.txt

```diff
@@ -192,7 +184,7 @@ geographiclib==2.0
     # via geopy
 geojson==3.1.0
     # via leafmap
-geopandas==0.14.4
+geopandas==1.0.0
```

Comment:
Ah - unrelated to python version, this was released yesterday!

@sf-dcp big news

Reply:
that's awesome!